scss/tools.scss

```diff
@@ -0,0 +1,16 @@
+.tools{
+  padding: 60px 100px;
```

Comment:
It's not a good practice assign this as padding because in the responsive you could have problems, so i recommend you change by `max-width: 100%; width: 1000px;`